src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +35,24 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * This method converts the set of keywords to a set of lower case strings and returns it.
+     * @return set of lower case string
+     */
+    public static Set<String> keywordToLowerCaseSet(Set<String> keywords) {
+        Set<String> lowerCaseKeywords = new HashSet<>();
+        for(String key : keywords) {
+            lowerCaseKeywords.add(key.toLowerCase());
+        }
+        return lowerCaseKeywords;
+    }
+
+    public static List<String> keywordToLowerCaseList(List<String> keywords) {
```

Comment:
Both methods can have the same same, if you apply `method overloading`. Here for more details: https://nus-cs2103-ay1718s2.github.io/website/book/common/outcomes.html